### PR TITLE
update-ols-2-grad-dates

### PR DESCRIPTION
**Pull request by yochannah**

### CODE REVIEW

**Comment by malvikasharan** [comment]
Looks good to me!

File: _data/ols-2-schedule.yaml [hunk 1]
```diff
@@ -874,12 +874,12 @@
   calls:
     -
       type: Cohort
-      duration: 90 min
+      duration: 60 min
       title: 'Final presentations & Graduation! - Group 1'
-      date: December 19, 2020
-      time:
+      date: 15 December, Tuesday
```
Comment (bebatut) [inline _data/ols-2-schedule.yaml:879]:
```suggestion
      date: December 15, 2020
```

File: _data/ols-2-schedule.yaml [hunk 2]
```diff
@@ -874,12 +874,12 @@
...
+      time: 09:00 - 10:00 AM UTC
```
Comment (bebatut) [inline _data/ols-2-schedule.yaml:880]:
```suggestion
      time: '10:00'
```

File: _data/ols-2-schedule.yaml [hunk 3]
```diff
@@ -894,17 +894,37 @@
       resources:
     -
       type: Cohort
-      duration: 90 min
+      duration: 60 min
       title: 'Final presentations & Graduation! - Group 2'
-      date: December 19, 2020
-      time:
+      date: 16 December, Wednesday
```
Comment (bebatut) [inline _data/ols-2-schedule.yaml:899]:
```suggestion
      date: December 16, 2020
```

File: _data/ols-2-schedule.yaml [hunk 4]
```diff
@@ -894,17 +894,37 @@
...
+      time: 14:00 - 15:00 UTC
```
Comment (bebatut) [inline _data/ols-2-schedule.yaml:900]:
```suggestion
      time: '15:00'
```

File: _data/ols-2-schedule.yaml [hunk 5]
```diff
@@ -894,17 +894,37 @@
...
+
+          Your presentation can be in the form of a demo, short presentation or a walk through your website
+      after: |
+        - Celebrate your graduation from OLS-2!
+      resources:
+       -
+      type: Cohort
+      duration: 60 min
+      title: 'Final presentations & Graduation! - Group 3'
+      date: 17 December, Thursday
```
Comment (bebatut) [inline _data/ols-2-schedule.yaml:919]:
```suggestion
      date: 17 December 17, 2020
```

File: _data/ols-2-schedule.yaml [hunk 6]
```diff
@@ -894,17 +894,37 @@
...
+      time: 17:30 - 18:30 UTC
```
Comment (bebatut) [inline _data/ols-2-schedule.yaml:920]:
```suggestion
      time: '18:30'
```

**Review by bebatut — COMMENTED** [review]
Made some small changes 😄

**Pull request merged**